build/npm/postinstall.js

```diff
@@ -33,4 +36,42 @@ const extensions = [
 	'html'
 ];
 
-extensions.forEach(extension => npmInstall(`extensions/${extension}`));
\ No newline at end of file
+extensions.forEach(extension => npmInstallSync(`extensions/${extension}`));
+
+function rmDuplicateModulesSync() {
+
+	function fetchModuleIds(basepath) {
+		const result = new Map();
+		for (const candidate of fs.readdirSync(path.join(basepath, 'node_modules'))) {
+			try {
+				let raw = fs.readFileSync(path.join(basepath, 'node_modules', candidate, 'package.json'));
+				let data = JSON.parse(raw);
+				result.set(data._id, path.join(basepath, 'node_modules', candidate));
+			} catch (e) {
+				if (e.code !== 'ENOENT') {
+					throw e;
+				}
+			}
+		}
+		return result;
+	}
+
+	const duplicates = new Set();
+	const baseModules = fetchModuleIds('');
+
+	for (const extension of extensions) {
+		const extensionModules = fetchModuleIds(`extensions/${extension}`);
+		for (let [key, value] of extensionModules) {
+			if (baseModules.has(key)) {
+				duplicates.add(value);
+			}
+		}
+	}
+
+	for (let duplicate of duplicates) {
+		console.log(`REMOVING duplicate module '${duplicate}'`);
+		rimraf.sync(path.join(process.cwd(), duplicate));
```

Comment:
unsure - we seems to rely with that being implicit anyways because everywhere we just use *cwd-relative* paths, e.g. when running `npm i`